Automattic/wp-calypso
# SSO: Show modal on disconnection

This PR adds `SurveyModal` component and uses that to display a modal when users disable sso

Related to #

## Proposed Changes

* Add SurveyModal component
* Wire it to be used in sso

## Testing Instructions
- Build the PR or use the live link
- Go to /settings/security/{someAtomicSite}
- Disable sso
- You should see a modal that leads to the sso survey



*

## Pre-merge Checklist
